### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dsyme**
The notebook server in the docker image requires token or jupyter password authentication.  Clarify the instructions

@cgravill Do the other approaches also require the jupyter notebook server token to be entered by the client?

### CODE REVIEW

**Comment by dsyme** [comment]
(@cgravill I removed the table of contents as it's really hard to keep up-to-date, is that ok?)

**Comment by cgravill** [comment]
Yes, there needs to be a token passed. Usually this is hidden as when you start via a `jupyter notebook` or equivalent it passes the token into the browser start so it seems hidden to the user.

**Comment by cgravill** [comment]
I don't feel strongly about the table to contents. Some new people expressed confusion - they didn't really know what Jupyter was just that they wanted to do things interactively. The table of contents approach seems to have worked well for https://github.com/giraffe-fsharp/Giraffe so I was trying it as a lightweight experiment.

I agree it's a bit of a faff but I didn't really expect this sudden influence of - very welcome - activity. :-)

**Comment by dsyme** [comment]
@cgravill Thanks - shame that the token is required, but I understand

It seems a port mapping is also always required when starting via docker, which surprised me, i.e. `-p 8888:8888` - is that expected?

**Comment by dsyme** [comment]
Re TOC - yes, understood.  @gbaydin and myself will keep refining the getting started experience to make it as simple as possible according to @gbaydin's recent experience with Python

BTW the docker images are pretty big, it will be good to eventually trim them a bit.

File: README.md [hunk 1]
```diff
@@ -58,7 +52,7 @@ Jupyter will start and a notebook with F# can be selected. This can be run via "
 
 If the launch fails in the console window, check that the Anaconda version used is currently added to the path. If not, uninstalling Anaconda and reinstalling using instructions 1-
 
-## Mac Manual Installation
+## OSX Local Installation and Execution
```
Comment (slang25) [inline README.md]: `macOS`

**Pull request merged**